test/modules/core/effects/lighting-effect.spec.ts

```diff
@@ -13,6 +13,16 @@ const testViewport = new MapView().makeViewport({
   viewState: {longitude: -122, latitude: 37, zoom: 13}
 });
 
+function makeMockContext(device, layerManager) {
+  return {
+    device,
+    deck: {
+      _addDefaultShaderModule: layerManager.addDefaultShaderModule.bind(layerManager),
```

Comment:
Could avoid this by doing: https://github.com/visgl/deck.gl/pull/8681#discussion_r1531718265